plugins/algoltek-usb/fu-algoltek-usb-device.c

```diff
@@ -0,0 +1,514 @@
+/*
+ * Copyright (C) 2023 Algoltek, Inc.
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include "fu-algoltek-usb-common.h"
+#include "fu-algoltek-usb-device.h"
+#include "fu-algoltek-usb-firmware.h"
+#include "fu-algoltek-usb-struct.h"
+
+struct _FuAlgoltekUsbDevice {
+	FuUsbDevice parent_instance;
+};
+
+G_DEFINE_TYPE(FuAlgoltekUsbDevice, fu_algoltek_usb_device, FU_TYPE_USB_DEVICE)
+
+static gboolean
+algoltek_device_ctrl_transfer(FuAlgoltekUsbDevice *self,
```

Comment:
I think in general all the functions in this file should start with `fu_algoltek_usb_device_` prefixes. I know it's verbose, but it makes debugging from a stacktrace much easier.

Reply:
Okay. We will check it. The original idea which we didn't use `fu_algoltek_usb_device_` prefix is to distinguish between generated function name and custome function name.

Reply:
Right; the reason we prefer the full prefix is that we're using the function name to "key" the startup hot-spots for profiling. Using the filename+function name is of course possible, but often redundant.

Reply:
Updated! All functions in device.c file start with `fu_algoltek_usb_device_` prefixes.